### ISSUE TITLE
[Bug/Feature Request]: Why the covers look like this?

### ISSUE THREAD

**Issue by anotherduckling** [ollm/OpenComic#168]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/ollm/OpenComic/issues) for a bug report that matches the one I want to file, without success.

### OpenComic Version

v1.0.0-beta.4

### Operating System

Windows 11

### Steps to reproduce

1. Open a folder with manga

### Expected Behavior

It should show banner with full view instead of showing it as gallery image

### Actual Behavior

![image](https://github.com/ollm/OpenComic/assets/57977673/39f426b6-d086-4f2a-8218-89d3bb4b9d93)

**Comment by ollm** [comment]
Is this bug related to local artwork assets? Or do you just want the first image in the folder to be as a poster?

If it's the latter, I plan to add an option in the next beta to do that.

**Comment by anotherduckling** [comment]
At first i took it as a bug. My bad. I think it should look like this.
![image](https://github.com/ollm/OpenComic/assets/57977673/3a4a2c31-90c8-4a17-b2a9-5b4f99cdf903)


Can u add a toggle so that the cbz file acts like a file instead of a folder view? 
![image](https://github.com/ollm/OpenComic/assets/57977673/1f1afb06-4027-4cab-872d-b0a0f22a5808)

Also, off-topic, the readme mentions "Tracking with sites" but I couldn't find it.

**Comment by ollm** [comment]
> At first i took it as a bug. My bad. I think it should look like this.

Currently, OpenComic shows a poster if there is an image in the cbz/folder with certain names (poster, cover, default, folder and series) or if has the same cbz/folder name, but if they don't exist, it shows the first 4 images, that's why I plan to add the option to show the first image of the folder if there are no more folders inside it.

Just in case,  could you send me one of the cbz to check that the poster does not have a different name? This is my e-mail oleguer.llopart.mora@gmail.com

> Can u add a toggle so that the cbz file acts like a file instead of a folder view?

I'm considering adding an option to open directly on the first page instead of the file list, currently it has the "Open directly in continue reading instead of the file list" which does practically the same thing (The option is in settings page)

> Also, off-topic, the readme mentions "Tracking with sites" but I couldn't find it.

The feature is found when reading (Currently only in AniList), for it to work correctly the folder structure must be as follows:

```
Library
├───Frieren
├───One Punch Man
```

And not like this: 

```
Library
├───Manga
│   ├───Frieren
│   └───One Punch Man
├───Comics
```

![Captura desde 2023-11-05 16-38-56](https://github.com/ollm/OpenComic/assets/18549191/d0ee43c5-86a5-42d9-9572-384596e1fc06)

![Captura desde 2023-11-05 16-39-16](https://github.com/ollm/OpenComic/assets/18549191/4a15fbdb-53b3-4dad-8a8c-99e904be7f34)

**Comment by anotherduckling** [comment]
i have toggles this on but still showing as image grid

![image](https://github.com/ollm/OpenComic/assets/57977673/08f73a7c-3540-4b7f-908b-47ac94626fac)
![image](https://github.com/ollm/OpenComic/assets/57977673/522f14e0-e0da-4bd9-9265-338d448e8064)

i think i need to read it like that first, then the toggle will select the previous read. In my opinion the image grid view doesnt look necessary. Also have sent the cbz.

**Comment by ollm** [comment]
> i have toggles this on but still showing as image grid, i think i need to read it like that first, then the toggle will select the previous read. In my opinion the image grid view doesnt look necessary.

I also plan to add an option (Actived by default) to open files directly without going through the explorer (cbz, pdf, epub, etc.)

> Also have sent the cbz.

I have checked the `cbz` file and the poster appears to be only the first image, numbered in `p000`, when I implement the function to show the first image as a poster it should already be displayed correctly. There is also a file with metadata about the manga (Name, author, etc), but OpenComic currently does not display this type of information.

I'm currently working on epub support, but will implement these features once basic epub support is finished.

**Comment by ollm** [comment]
The latest beta version is now available with the 2 features implemented ([v1.0.0-beta.5](https://github.com/ollm/OpenComic/releases))

- Option to use the first image in the folder/file as a poster a5cf998786b17e451ce7d1a8fa24ac287779de44
- Option to open folder/file directly in first image or in continue reading 6a20160bcd96291c14d16d025baf7aef7ebe13c9

Feel free to comment if you find any bugs.

**Comment by ChameleonScales** [comment]
Any chance this could be made to work such that if there is only one image displayed in the poster, it would always cover the whole poster instead of only one quarter?
For example, currently if a folder or archive contains only one subfolder, the poster shows the first image in the first quarter and leaves the rest blank.
